tutorials/convert_lit_models.md

```diff
@@ -5,34 +5,55 @@ Several models retrieved from external sources must be reformatted with naming c
 
 ### Full Finetuning
 
-After full finetuning, your checkpoint directory will contain a file named `lit_model_finetuned.pth`and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After full finetuning, your checkpoint directory will contain a file named `lit_model_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
+    --checkpoint_name lit_model_finetuned.pth  \
+    --out_dir out/full/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
 ```
 
 ### Adapter and Adapter V2 Finetuning
 
-After finetuning with either Adapter technique, your checkpoint directory will contain a file named `lit_model_adapter_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After finetuning with either Adapter technique, your checkpoint directory will contain a file named `lit_model_adapter_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
+
+### Adapter
+
+```sh
+python scripts/convert_lit_checkpoint.py \
+    --checkpoint_name lit_model_adapter_finetuned.pth  \
+    --out_dir out/adapter/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
+```
+
+### Adapter V2
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_adapter_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
+    --checkpoint_name lit_model_adapter_finetuned.pth  \
+    --out_dir out/adapter_v2/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
 ```
 
 ### LoRA Finetuning
 
-After finetuning with LoRA, your checkpoint directory will contain a file named `lit_model_lora_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After finetuning with LoRA, your checkpoint directory will contain a file named `lit_model_lora_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_lora_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
+    --checkpoint_name lit_model_lora_finetuned.pth  \
+    --out_dir out/lora/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
 ```
+
+### Submitting Issues
+
+If you run into errors when using convert_lit_checkpoint, please [submit an issue](https://github.com/Lightning-AI/lit-gpt/issues) in GitHub.
```

Comment:
Doesn't this apply to all sorts of issues?

Reply:
removed in [7cbea36](https://github.com/Lightning-AI/lit-gpt/pull/362/commits/7cbea36843faf324908441d6fc08faf86e39259b)